operator/src/test/java/oracle/kubernetes/operator/logging/LoggingFacadeTest.java

```diff
@@ -0,0 +1,193 @@
+// Copyright 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Licensed under the Universal Permissive License v 1.0 as shown at
+// http://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator.logging;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.*;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.junit.Test;
+
+public class LoggingFacadeTest {
+
```

Comment:
Every test seems to invoke:
```
MockLogger mockLogger = new MockLogger();
LoggingFacade loggingFacade = new LoggingFacade(mockLogger);
```
That suggests that those should be fields on the test, which get initialized in their definitions. That reduces code duplication.

Also, MockLogger isn't actually a mock. See https://blog.pragmatists.com/test-doubles-fakes-mocks-and-stubs-1a7491dfa3da